### PR TITLE
Ways to Run page

### PR DESCRIPTION
**Pull request by patch0**
First pass at adding the restart pages.  Still need to add in content.

### CODE REVIEW

**Comment by grega** [comment]
Content added. Needs images adding, along with confirmation of all links and also quote copy.

File: components/ways-to-run/Hero.vue [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      <p>CoderDojo volunteers and clubs all around the world have been affected by the coronavirus pandemic. Champions and mentors from across the globe have talked to us to share how they are running their clubs now, and we're amazed by their resilience and creativity.</p>
+
+      <p>Inspired by these community examples, here are some ways you can continue to provide fun, creative learning opportunities for the young people and volunteers in your Dojo.</p>
+
+      <p>Choose the option that works best for you, depending on your current situation and your Dojo members' needs, and feel free to adapt your club activities if your situation and local health guidance change.</p>
+    </div>
+    <div class="c-hero__step-container">
+      <div class="c-hero__step-cards">
+         <div class="c-hero__step-card">
+            <img src="~/static/images/ways-to-run/ways-red.jpg" class="c-hero__step-image">
```
Comment (glenpike) [inline components/ways-to-run/Hero.vue:14]: Missing alt tag - can use `alt=""` as this is decorative image.
Check other images have correct alt tags too.

**Review by glenpike — CHANGES_REQUESTED** [review]
Alt tags please

File: components/ways-to-run/Hero.vue (hunk 1 above)
Comment (grega) [reply components/ways-to-run/Hero.vue:14]: Thanks v.much, they were on the to-do list but ended up getting swamped by other things :|

Comment (glenpike) [reply components/ways-to-run/Hero.vue:14]: Still missing? (Same with yellow and green?)

**Review by glenpike — APPROVED** [review]
LGTM

**Pull request merged**